traveler-ios-ui/TravelerKitUI/ViewControllers/SimilarItemsViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SimilarItemsViewController.swift
+//  TravelerKitUI
+//
+//  Created by Rakin Hoque on 2019-11-01.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+public protocol SimilarItemsViewControllerDelegate: class {
+    func collapseSimilarItemsView()
```

Comment:
This is a bad name for this delegate function. We can't assume that the person implementing this screen wants to collapse the SimilarItemsVC because there's no items. Our approach here is to delegate back to the parent VC the expected behavior. We also always pass back the controller. In this case the function should be called something around similarItemsViewController(_ controller: SimilarItemsViewController, didFailWith: Error)